engine/src/main/java/org/terasology/rendering/opengl/AbstractFBOsManager.java

```diff
@@ -44,8 +44,8 @@
  * rendering engine.
  * <p>
  * Default FBOs:
- * sceneOpaque:  Primary FBO: most visual information eventually ends up here
- * sceneOpaquePingPong:  The sceneOpaque FBOs are swapped every frame, to use one for reading and the other for writing
+ * writeOnlyGBuffer:  Primary FBO: most visual information eventually ends up here
+ * readOnlyGBuffer:  The gbuffer FBOs are swapped around when needed, to use one for reading and the other for writing
```

Comment:
By the way, the sceneBloom# fbos below can be described as:

`A number of buffers used to repeatedly blur and downsample the content of the high pass fbo. The resulting blurred version of the high pass is then used for the bloom effect. `

And there is no more ocUndistorted fbo, right?

And we should probably remove all those "scene" terms.